tests/components/fronius/test_sensor.py

```diff
@@ -70,7 +24,7 @@ def assert_state(entity_id, expected_state):
     mock_responses(aioclient_mock, night=True)
     await setup_fronius_integration(hass)
 
-    assert len(hass.states.async_all()) == 56
+    assert len(hass.states.async_all()) == 55
```

Comment:
there was an additional `persistent_notification.config_entry_discovery` state from the yaml import

Reply:
We can limit the states by passing the sensor domain to `hass.states.async_all`. This will make the test more robust.

https://github.com/home-assistant/core/blob/70f43a14156566162d073b3e029d39b956b58b4c/homeassistant/core.py#L1072

Reply:
Cool! 👍